### ISSUE TITLE
Allow editing sent messages

### ISSUE THREAD

**Issue by bgilbert** [zulip/zulip-android#373]
Once a message is sent, there seems to be no way to modify it without switching to the web interface.  This compounds the damage after #179 strikes.  Perhaps an edit command could be added to the message long-press menu.

**Comment by timabbott** [comment]
Thanks for the feedback @bgilbert!  This is already done in master, so this feature will be in the next release (probably in a couple days)!  See #290.

**Comment by bgilbert** [comment]
Awesome, looking forward to it!

**Comment by bgilbert** [comment]
Awesome, looking forward to it!
